src/types.ts

```diff
@@ -0,0 +1,26 @@
+/*
+Copyright 2019 by Jakub Kaluzka
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/**
+ * The different status of a Cloudbuild job in an enumeration.
+ */
+
+export enum JobStatus {
```

Comment:
Since the `JobStatus` enumeration is strongly linked to the `Job` class (in terms of functional domain),  I don't really think that it's really useful to create a dedicated `types` file for it. 
`JobStatus` serves here more to avoid string literals in the `Job` class.

Reply:
I think that every interface, enum or type is somehow strongly related to a class/implementation. If you strongly insist I can bring it back, but I consider keeping types separated from implementation details as a good practice.

Reply:
Ok, got your point on this. We can keep JobStatus separated from the Job class.

In the OOP version, I'll make the JobStatus available in a `datas/` package (see https://github.com/an0rak-dev/vscode-gcp-cloudbuild/pull/12/files#diff-df36af871a3a35efe3088030031f5105R5 )

We can keep your changes here 😄